Sky Atlas/Sky Atlas.html

```diff
@@ -21703,6 +21982,18 @@ <h1>Sections & Primary Docs</h1>
           <td>Genesis Capital Backstop - Genesis Capital</td>
           <td>Core</td>
           <td>The Genesis Capital of an Agent is the lesser of (1) the amount of capital contributed by Sky to the Agent and (2) the total capital of the Agent.</td>
+        </tr>
+         <tr>
```

Comment:
remove single space to correct indentation

```suggestion
        <tr>
```